src/views/ipv/page-multiple-doc-check.njk

```diff
@@ -33,7 +33,7 @@
                 divider: 'pages.pageMultipleDocCheck.content.formRadioButtons.separateOptionsInFormText' | translate
               },
               {
-                value: "end",
+                value: "next/claimed-identity",
```

Comment:
Also can you make it just "claimedIdentity" and feature flagged as this is only available in dev at the moment.